lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -1095,6 +1095,14 @@ def test_axes3d_rotated():
     ax.view_init(90, 45, 0)  # look down, rotated. Should be square
 
 
+@mpl3d_image_comparison(['axes3d_roll.png'],
```

Comment:
We have nonzero roll angles in a few other tests, so I don't think we need an additional one here. (test_shared_view, test_bar3d_shaded)

Reply:
When you remove the test image, you'll have to squash your commits otherwise the "PR Cleanliness" check will complain about an image being added and then removed in the same PR.

Reply:
Ah good, I took out the superfluous roll test, combined the mouse rotation tests, and force pushed the result.